integration-tests/test/test_unbond.py

```diff
@@ -105,10 +142,15 @@ def test_unbond_validator(command_line_options: CommandLineOptions, random_gener
         wait_for_node_sees_block(context, bootstrap_node, b8)
 
         # block number 9
-        validator_1_balance = get_vault_balance(context, bootstrap_node, VALIDATOR_KEY_1.get_public_key().get_rev_address(), VALIDATOR_KEY_2, 100000, 1)
-        assert validator_1_balance < 20000000
+        validator_1_balance_before_refund = get_vault_balance(context, bootstrap_node,
+                                                              VALIDATOR_KEY_1.get_public_key().get_rev_address(),
+                                                              VALIDATOR_KEY_2, 100000, 1)
+        assert validator_1_balance_before_refund < validator_1_initial_wallet_amount
 
         # block number 10
         # withdraw happen in block number 9, result get in block 10
-        validator_1_balance = get_vault_balance(context, bootstrap_node, VALIDATOR_KEY_1.get_public_key().get_rev_address(), VALIDATOR_KEY_2, 100000, 1)
-        assert validator_1_balance > 50000000
\ No newline at end of file
+        # get the bonding amount and the reward amount
+        validator_1_balance = get_vault_balance(context, bootstrap_node,
+                                                VALIDATOR_KEY_1.get_public_key().get_rev_address(), VALIDATOR_KEY_2,
+                                                100000, 1)
+        assert validator_1_balance == validator_1_balance_before_refund + validator_1_initial_bonding_amount + rewards_of_v1
```

Comment:
Why should
`validator_1_balance == validator_1_balance_before_refund + validator_1_initial_bonding_amount + rewards_of_v1` 
be true as opposed to
`validator_1_balance == validator_1_balance_after_refund + validator_1_initial_bonding_amount + rewards_of_v1`?